migrations/1_initial_migration.js

```diff
@@ -57,19 +84,19 @@ async function deployErc1820(account) {
 }
 
 async function deployDssDeploy(deployer) {
-	const vatFab = await deployer.deploy(artifacts.require('VatFab'))
-	const jugFab = await deployer.deploy(artifacts.require('JugFab'))
-	const vowFab = await deployer.deploy(artifacts.require('VowFab'))
-	const catFab = await deployer.deploy(artifacts.require('CatFab'))
-	const daiFab = await deployer.deploy(artifacts.require('DaiFab'))
+	const vatFab     = await deployer.deploy(artifacts.require('VatFab'))
+	const jugFab     = await deployer.deploy(artifacts.require('JugFab'))
+	const vowFab     = await deployer.deploy(artifacts.require('VowFab'))
+	const catFab     = await deployer.deploy(artifacts.require('CatFab'))
+	const daiFab     = await deployer.deploy(artifacts.require('DaiFab'))
```

Comment:
I will murder you in your sleep for this.

Reply:
hah, the spacing? I should really just create an array and map it...